interface/forms/vitalsM/table.sql

```diff
@@ -21,4 +21,4 @@ CREATE TABLE IF NOT EXISTS `form_vitalsM` (
 `head_circ`         FLOAT(5,2)      default 0,
 `oxygen_saturation` FLOAT(5,2)      default 0,
 PRIMARY KEY (id)
-) ENGINE=MyISAM;
+) ENGINE=InnoDB;
```

Comment:
Here are the files that would need to be changed:
contrib/forms/hp_tje_primary/table.sql:) ENGINE=MyISAM AUTO_INCREMENT=1 ;
contrib/forms/hp_tje_primary/table.sql:) ENGINE=MyISAM AUTO_INCREMENT=1 ;
contrib/forms/hp_tje_primary/table.sql:) ENGINE=MyISAM AUTO_INCREMENT=1 ;
contrib/forms/hp_tje_primary/table.sql:) ENGINE=MyISAM AUTO_INCREMENT=1 ;
contrib/forms/football_injury_audit/table.sql:) ENGINE=MyISAM;
contrib/forms/sports_fitness/table.sql:) ENGINE=MyISAM;
contrib/forms/contacts/table.sql:) ENGINE=MyISAM;
contrib/forms/vision/table.sql:) ENGINE=MyISAM;
contrib/forms/scanned_notes/table.sql:) ENGINE=MyISAM;
contrib/forms/assessment_intake/table.sql:) ENGINE=MyISAM;
contrib/forms/clinical_notes/table.sql:) ENGINE=MyISAM;
contrib/forms/hist_exam_plan/table.sql:) ENGINE=MyISAM;
contrib/forms/prosthesis/table.sql:) ENGINE=MyISAM;
contrib/forms/ippf_srh/table.sql:) ENGINE=MyISAM;
contrib/forms/body_composition/table.sql:) ENGINE=MyISAM;
contrib/forms/reviewofs/table.sql:) ENGINE=MyISAM;
contrib/forms/ped_pain/table.sql:) ENGINE=MyISAM;
contrib/forms/soccer_injury/table.sql:) ENGINE=MyISAM;
contrib/forms/strength_conditioning/table.sql:) ENGINE=MyISAM;
contrib/forms/review_of_systems/table.sql:) ENGINE=MyISAM;
contrib/forms/review_of_systems/table.sql:) ENGINE=MyISAM;
contrib/forms/psychiatrySet/brief_aan_verwijzer/table.sql:) ENGINE=MyISAM AUTO_INCREMENT=2 ;
contrib/forms/psychiatrySet/psychiatrisch_onderzoek/table.sql:) ENGINE=MyISAM AUTO_INCREMENT=2 ;
contrib/forms/psychiatrySet/intakeverslag/table.sql:) ENGINE=MyISAM AUTO_INCREMENT=2 ;
contrib/forms/psychiatrySet/intakeverslag/autosave.sql:) ENGINE=MyISAM;
contrib/forms/cricket_injury_audit/table.sql:) ENGINE=MyISAM;
contrib/forms/evaluation/table.sql:) ENGINE=MyISAM;
contrib/forms/evaluation/table.sql:) ENGINE=MyISAM;
contrib/forms/formmaker/formscript.pl:) ENGINE=MyISAM;
contrib/forms/individual_treatment_plan/table.sql:) ENGINE=MyISAM;
contrib/forms/prior_auth/table.sql:) ENGINE=MyISAM;
contrib/forms/specialist_notes/table.sql:) ENGINE=MyISAM;
contrib/forms/treatment_protocols/table.sql:) ENGINE=MyISAM;
contrib/forms/obstetrical/table.sql:) ENGINE=MyISAM;
contrib/forms/ped_fever/table.sql:) ENGINE=MyISAM;
contrib/forms/pain/table.sql:) ENGINE=MyISAM;
contrib/forms/example2/table.sql:) ENGINE=MyISAM;
contrib/forms/example/SQL.php:	) ENGINE=MyISAM
contrib/forms/ped_GI/table.sql:) ENGINE=MyISAM;
contrib/forms/hearingtest/table.sql:) ENGINE=MyISAM;